.github/ISSUE_TEMPLATE/---report-a-bug.md

```diff
@@ -0,0 +1,40 @@
+---
+name: "\U0001F41B Report a bug"
+about: Create a report to help us fix bugs
+title: ''
+labels: bug
+assignees: drahnr
+
+---
+
+**Describe the bug**
+
+<!-- A clear and concise description of what the bug is. -->
+
+**To Reproduce**
+
+Steps to reproduce the behaviour:
+
+1. Setting up preconditions with `...`
+2. Run `cargo ...`
+3. Execute `...`
+4. ...
+
+**Expected behavior**
+
+<!-- A clear and concise description of what you expected to happen. -->
+
+**Screenshots**
+
+<!-- If applicable, add screenshots or copies of the commandline output to help explain your problem.
+Use code blocks -->
+
+**Please complete the following information:**
+ - System: <!-- Fedora, Ubuntu, Win10, MacOS, ... with a version -->
+ - Obtained: <!-- git, cargo, binary -->
+ - Version: <!-- must contain the version of relevant backends, i.e. cuda and opencl -->
```

Comment:
We need their CUDA version, CUDNN version, and BLAS variant.

Reply:
If they use openCL we might as well ask for that too

Reply:
Not sure how relevant `OpenCL` is currently, but added it anyways :)